README.md

```diff
@@ -459,7 +534,7 @@ test('updating-time updates', function(assert) {
   return wait()
     .then(() => {
       fakeNow = new Date(2017);
-      pollTaskFor('updating-time#updateTime');
+      pollTaskFor(this._pollToken);
```

Comment:
FWIW, I still think user provided “labels” are better here 😃

I’m not going to try to convince you, but I do ask that we ensure both are documented....

Reply:
Bleh. Concede...